src/grammar.pest

```diff
@@ -1,97 +1,217 @@
 // pest. The Elegant Parser
 // Copyright (c) 2018 Dragoș Tiselice
-//
+// 
 // Licensed under the Apache License, Version 2.0
 // <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT
 // license <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. All files in the project carrying such notice may not be copied,
 // modified, or distributed except according to those terms.
-
-grammar_rules = _{ SOI ~ grammar_rule+ ~ EOI }
-
+//! Pest meta-grammar
```

Comment:
I'm thinking whether there's a way to have this without copy-pasting the meta grammar?

Reply:
I will find a way for that later.

There is nothing must to do, so I think we can merge and publish a version now.